creation/web_base/condor_platform_select.sh

```diff
@@ -196,7 +197,7 @@ for version_el in $(echo "$condor_version" | tr ',' ' '); do
             condor_platform="${version_el}-${os_el}-${arch_el}"
             condor_platform_id="CONDOR_PLATFORM_$condor_platform"
 
-            condor_platform_check=`grep "^$condor_platform_id " "$glidein_config" | awk '{print $2}'`
+            condor_platform_check=$(gconfig_get $condor_platform_id "$glidein_config")
```

Comment:
quote variable `"$condor_platform_id"` to protect for spaces

Reply:
done